packages/rocket-webhook-azure-infrastructure/src/synth/terraform-api-management-api-operation.ts

```diff
@@ -14,19 +15,21 @@ export class TerraformApiManagementApiOperation {
     endpoint: string
   ): apiManagementApiOperation.ApiManagementApiOperation {
     const apiManagementApi: apiManagementApi.ApiManagementApi = applicationSynthStack.apiManagementApi!
-    const idApiManagementApiOperation = utils.toTerraformName(appPrefix, `amaor${endpoint}`)
+    const suffix = UUID.generate().toString().substring(0, 23)
+    const idApiManagementApiOperation = utils.toTerraformName(appPrefix, suffix)
 
+    const operationId = `${endpoint.replace('/', '-')}POST`
```

Comment:
Question: Could we allow the user to specify as endpoint config the HTTP method too? That way, we would allow defining GET endpoints, which can be useful when legacy services that don't support GraphQL interact with Booster.

Reply:
It's not supported right now. I included it in the TODO list